src/libraries/Common/src/System/HexConverter.cs

```diff
@@ -169,6 +243,21 @@ public static bool TryDecodeFromUtf16(ReadOnlySpan<char> chars, Span<byte> bytes
 
             int i = 0;
             int j = 0;
+            if (Avx2.IsSupported && bytes.Length > 32)
+            {
+                unsafe
```

Comment:
I believe the convention here is to put `unsafe` to the signature and don't use "inline-blocks".